recipes/mongo-c-driver/all/conanfile.py

```diff
@@ -0,0 +1,155 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class MongoCDriverConan(ConanFile):
+    name = "mongo-c-driver"
+    license = "Apache License 2.0"
```

Comment:
We use the "short identifier" https://spdx.org/licenses/Apache-2.0.html
```suggestion
    license = "Apache-2.0"
```

Reply:
Fixed